examples/ligand_in_water/ligand_in_water.ipynb

```diff
@@ -36,8 +36,7 @@
     "from openff.units import unit\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from openff.interchange.interop.openmm import to_openmm_positions
</pre><p> </p><p>I think we should encourage users to import from public modules.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-interchange/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Oops yep, good catch - forgot to update this one in the other PR.</p>